contents/tutorials/ios-ab-tests.md

```diff
@@ -0,0 +1,195 @@
+
+Next, configure your PostHog instance in your `App.swift` struct initializer:
+
+```swift
+// in App.swift
+import SwiftUI
+import PostHog
+
+@main
+struct App: App {
+    init() {
+        let configuration = PHGPostHogConfiguration(apiKey: "<ph_project_api_key>", host: "<ph_instance_address>")
+        configuration.captureApplicationLifecycleEvents = true
+        configuration.recordScreenViews = true
+        PHGPostHog.setup(with: configuration)
+    }
+    
+    var body: some Scene {
+        WindowGroup {
+            ContentView()
+        }
+    }
+}
+```
+
+To get your PostHog API key and host, [sign up to PostHog](https://app.posthog.com/signup). Then, you can find your API key and host in your [project settings](https://app.posthog.com/settings/project). Replace the strings `<ph_project_api_key>` and `<ph_instance_address>` with your API key and host.
+
+Finally, [capture a custom event](/docs/libraries/ios#capturing-events) when the button on `FeatureScreenView` is clicked. We'll use this event as our goal metric in our A/B test.
+
+```swift
+// in FeatureScreenView.swift
+import PostHog
+
+// ...rest of code
+
+Button("Click Me!") {
+    PHGPostHog.shared()?.capture("feature_button_clicked")
+}
+
+// ...rest of code
+```
+
+To check your setup, build and run your app. Click your button a few times. You should start seeing events in the [PostHog event explorer](https://app.posthog.com/events).
+
+![iOS events captured](../images/tutorials/ios-ab-tests/event-captured.png)
+
+## Create an A/B test in PostHog
+
+The next step is to set up an A/B test (we call them experiments in PostHog).
+
+Go to the [Experiments tab](https://app.posthog.com/experiments) in PostHog and click "New experiment". Add the following details to your experiment:
+
+1. Name it "iOS background color experiment".
+2. Set "Feature flag key" to `ios-background-color-experiment`.
+3. Under the experiment goal, select the `feature_button_clicked` we created in the previous step.
+4. Use the default values for all other fields.
+
+Click "Save as draft" and then click "Launch".
+
+![Experiment setup in PostHog](../images/tutorials/ios-ab-tests/experiment-setup.png)
+
+## Implement the A/B test in Xcode
+
+The final step is to add the experiment code. We'll add code that does the following:
+
+1. Fetch the `ios-background-color-experiment` flag.
+2. Change the background color of `FeatureScreenView` based on the value of the flag (`control` or `test`).
+
+To do this, update the code in `ContentView.swift` with the following:
+
+```swift
```

Comment:
I wonder if it makes sense to keep the whole code snippet rather than the omission, right now the indentation is strange and a novice developer might have difficulties finding out where to place the code/how to read it.

Reply:
I think I'll include the function name, but would prefer to avoid to include all the code (less clear this way)